### ISSUE TITLE
Object Reference Error for ASP.NET controller actions with Json(Method())

### ISSUE THREAD

**Issue by someguy20336** [someguy20336/TypeRight#67]
The following controller action will crash in ASP.NET, starting around v0.12.

```csharp
public JsonResult Action() 
{
    return Json(CallAMethod());
}
```

This is because `IgnoreActionResultMethodNodeHandler` only had the AspNetCore `ActionResult` class, thus causing a null reference exception in the `HasBaseType` extension used in that class.

This is pretty bad, so I will likely patch pre v1.0.

**Comment by someguy20336** [comment]
published visx v1.0.2 in hopes for some more insight on the crash (just has a stack trace)

**Comment by someguy20336** [comment]
Also fixed in v0.13.1
